server/index.js

```diff
@@ -37,6 +37,30 @@ app.get('/products/:product_id', (req, res) => {
     });
 });
 
+app.get('/reviews', (req, res) => {
+  const id = req.query.product_id;
+  axios.get(`${API_BASE_URL}/reviews?product_id=${id}`, options)
+    .then((response) => {
+      res.send(response.data);
```

Comment:
do you want to send a status code back as well?

Reply:
I think it needs to send status code everytime